content/blog/2022/01/multiplatform-kubewarden.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Multiplatform Kubewarden
+authors:
+    - Rafael Fernández López
+date: 2022-02-01
+---
+
+The Kubewarden team is glad to announce that in the spirit of helping
+Policy Authors and Cluster Administrators, the project is now
+officially multiplatform.
+
+The list of supported platforms as of now are:
+
+- Policy Server, as [container image](https://github.com/kubewarden/policy-server/pkgs/container/policy-server):
+  - `linux/amd64` (with the `musl` libc)
+  - `linux/arm64` (with the `musl` libc)
+
+- kwctl, as a [standalone binary](https://github.com/kubewarden/kwctl/releases):
+  - `darwin` (`x86_64`)
+  - `linux` (`aarch64`, with the `musl` libc)
+  - `linux` (`x86_64`, with the `musl` libc)
+  - `windows` (`x86_64`, with `MSVC`)
```

Comment:
As far as I know amd64 is equivalent to x86_64. Can we use the same name for both policy server and kwctl?

Reply:
Yeah... it's that docker names it as `amd64`, and our main artifact of the `policy-server` is a container image...

Reply:
Same happens with `arm64` vs `aarch64`. I favour `aarch64` in the binary because is what we use, while for the container image is `arm64`.